### PR TITLE
fix(composer): enable convert_unsafe_embeds to fix security issue

### PR DESCRIPTION
**Pull request by beawar**
Fixes security issue 65

### CODE REVIEW

**Review by nubsthead — APPROVED** [review]

**Review by CataldoMazzilli — APPROVED** [review]

**Comment by beawar** [comment]
I'll wait for an additional approval from @gnekoz @giuliano176 or @dhavaldodiya to be sure the change is ok also on mails side

**Comment by beawar** [comment]
Ping @gnekoz @giuliano176 @dhavaldodiya

**Review by yukiLu6 — APPROVED** [review]

**Review by dhavaldodiya — APPROVED** [review]

**Pull request merged**
